array-map/main.js

```diff
@@ -0,0 +1,23 @@
+const { number } = require("assert-plus");
+
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+let doubled = numbers.map(num => num*2)
+console.log(doubled)
+
+let prices = numbers.map(num => `$${num}.00`)
```

Comment:
While this works here it would be better to use `$${number.toFixed(2)}`, that will account for if the number already has a decimal.